release/spire-extras/conf/oidc-discovery-provider/oidc-discovery-provider.conf

```
@@ -0,0 +1,11 @@
+log_level = "debug"
+domain = "public.example.org"
+acme {
+    cache_dir = "/tmp/.acme-cache"
+    # If you accept the ACME service Terms of Service,
+    # Uncomment the line below in order to work
+    # tos_accepted = true
```

Thanks for adding this comment! Can I make a small suggestion regarding the language?
```suggestion
    # Uncomment the line below to accept the ACME provider Terms of Service. This is
    # required to use ACME to provide certificates for the OIDC discovery provider. 
    # tos_accepted = true
```

---

@azdagron Thanks for the suggestion. I updated comment with https://github.com/spiffe/spire/pull/1424/commits/b2add7ae838d21557f1bc4fce6d2413daaef4315 .